### ISSUE TITLE
how to use the directive of EllipsisTooltip

### ISSUE THREAD

**Issue by zhleicd** [IBM/carbon-components-angular#940]
## Detailed description
Hi team:
I tried to use the directive of EllipsisTooltip in my angular project, 
but **unexpected error happened as below:**
ERROR Error: Uncaught (in promise): Error: Template parse errors:
Can't bind to 'ibmEllipsisTooltip' since it isn't a known property of 'div'. ("
        </ng-template>
        <div class="l2c-collection-insight-card-content-value" 
          [ERROR ->][ibmEllipsisTooltip]="actualTooltip" trigger="hover" [placement]="'bottom'">
          <span style="f"): ng:///CollectionModule/CollectionInsightCardComponent.html@23:10
Can't bind to 'ibmEllipsisTooltip' since it isn't a known property of 'div'. ("
        </ng-template>
        <div class="l2c-collection-insight-card-content-value" 
          [ERROR ->][ibmEllipsisTooltip]="actualTooltip" trigger="hover" [placement]="'bottom'">
          <span style="f"): ng:///CollectionModule/CollectionInsightCardComponent.html@23:10

**my source is as below:**
![image](https://user-images.githubusercontent.com/39113573/70047250-a60f7d00-1602-11ea-8497-f47ebce12565.png)

but when I changed [ibmEllipsisTooltip] to [ibmTooltip], it worked normally.
Would you please show me a demo about how to use the directive of EllipsisTooltip.

## Additional information
version of carbon-components: 10.4.1
version of carbon-components-angular: 3.13.4

**Comment by cal-smith** [comment]
I can't see a reason why it wouldn't work ... that said, the ellipsis tooltip relies on the fairly specific styles, so it's not likely to work in your app - you may be better off using the standard tooltip

**Comment by zhleicd** [comment]
I think I set right style in div, it will show ... when text overflows. What I want to do is to show tooltip when text overflows and I can't find a demo in storybook about how to use EllipsisTooltip, I tried to use it by imitating the demo of 'Tooltip with no icon' but error 'Can't bind to 'ibmEllipsisTooltip' since it isn't a known property of 'div'' occurred.
So I think maybe I didn't use EllipsisTooltip correctly, I want to get a demo on how to use it.
